src/pages/Admin/constants.ts

```diff
@@ -40,3 +42,173 @@ export const proposalShape: SchemaShape<ProposalBase> = {
   title: stringByteSchema(4, 64),
   description: stringByteSchema(4, 1024),
 };
+
+type SIDEBAR_ROUTES = Exclude<
+  adminRoutes,
+  adminRoutes.proposal | adminRoutes.templates
+>;
+
+const _to: keyof Link = "to";
+
+export const LINKS: { [key in SIDEBAR_ROUTES]: Link & { [_to]: key } } = {
+  [adminRoutes.index]: {
+    title: "Dashboard",
+    to: adminRoutes.index,
+    icon: {
+      type: "Dashboard",
+      size: 24,
+    },
+    end: true,
+  },
+  [adminRoutes.withdraws]: {
+    title: "Withdraw",
+    to: adminRoutes.withdraws,
+    icon: {
+      type: "MoneyBill",
+      size: 24,
+    },
+  },
+  [adminRoutes.contributions]: {
+    title: "Contributions",
+    to: adminRoutes.contributions,
+    icon: {
+      type: "DollarCircle",
+      size: 24,
+    },
+  },
+  [adminRoutes.invest]: {
+    title: "Invest Dashboard",
+    to: adminRoutes.invest,
+    icon: {
+      type: "Analytics",
+      size: 24,
+    },
+  },
+  [adminRoutes.settings]: {
+    title: "Settings",
+    to: adminRoutes.settings,
+    icon: {
+      type: "PermDataSettings",
+      size: 24,
+    },
+  },
+  [adminRoutes.edit_profile]: {
+    title: "Edit Profile",
+    to: adminRoutes.edit_profile,
+    icon: {
+      type: "User",
+      size: 24,
+    },
+  },
+  [adminRoutes.proposals]: {
+    title: "Decision Center",
+    to: adminRoutes.proposals,
+    icon: {
+      type: "ClipboardCheck",
+      size: 24,
+    },
+  },
+  [adminRoutes.widget_config]: {
+    title: "Widget Configuration",
+    to: adminRoutes.widget_config,
+    icon: {
+      type: "Widget",
+      size: 24,
+    },
+  },
+};
+
+// THESE WILL BE THE APPROPRIATE ICONS TO USE FOR FUTURE PAGES
+// TODO: REMOVE ONCE ALL ARE MOVED INTO `LINKS` ABOVE.
```

Comment:
Left this here as it will be helpful once we get to creating/migrating the pages with icons/URLs defined within.